Tests/MongoSwiftSyncTests/MongoDatabaseTests.swift

```diff
@@ -175,6 +175,75 @@ final class MongoDatabaseTests: MongoSwiftTestCase {
         expect(listNamesEvents[2].command["nameOnly"]).to(equal(false))
         expect(listNamesEvents[3].command["nameOnly"]).to(equal(false))
     }
+
+    func testAggregate() throws {
+        let client = try MongoClient.makeTestClient()
+        // $currentOp must be run on the admin database
+        let db = client.db("admin")
+        let result = try db.aggregate([["$currentOp": ["allUsers": true]]]).all()
+        let op = result.filter {
```

Comment:
instead of `filter { ... } [0]`, you can use [`first(where:)`](https://developer.apple.com/documentation/swift/array/1848165-first) which performs the same purpose in a single method.